src/server/start.js

```diff
@@ -0,0 +1,38 @@
+const { parse } = require('url')
+
+const bodyParser = require('body-parser')
+const express = require('express')
+
+const { requestLogger } = require('./logger')
+
+function startServer(app) {
```

Comment:
This is considered as `server` and the functionality `startServer`.

Although we start an _express server`, I wonder if this should be `api/reloader.js`.

Maybe a better understanding of what happens here could help me define this more accurately.

1. we create an express server
2. we create a `reloadRoute` that, for any request received, sends a reload message to the dev server
3. we expose a custom endpoint `/_next_reload` that triggers our `reloadRoute`


@jescalan not sure I get this one below

```js
// handle all other routes with next.js
  server.all('*', (req, res) => handle(req, res, parse(req.url, true)))
```

we start both a _next dev server` and this _express server`, how are those two exactly related?

If this was just a custom endpoint having 
```js
server.use('/__next_reload', reloadRoute)
```
would suffice.

However, it seems we are doing a passthrough to all other requests to this express server through `server.all('*', (req, res) => handle(req, res, parse(req.url, true)))`